test/configCases/url/defaults/file.svg

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
How can I dedupe this `fixtures` for the tests, would

```
|–url
||– file.png
||– file.svg
||– case
|||– index.js
|||– webpack.config.js 
```

work?

Reply:
You can create a directory with underscore to store stuff: `url/_assets/...`